Src/BootCamp.Chapter/BinaryConverter.cs

```diff
@@ -6,14 +6,45 @@ namespace BootCamp.Chapter
 {
     public static class BinaryConverter
     {
+        private const int OnBit = 1;
+        private const int OffBit = 0;
+
         public static long ToInteger(string binary)
         {
-            return 0;
```

Comment:
Originall I only had:
`        public static long ToInteger(string binary)
        {
            var integer = 0;

            if (!string.IsNullOrEmpty(binary))
            {
                try
                {
                    integer = Convert.ToInt32(binary, 2);
                }
                catch (Exception)
                {
                    throw new InvalidBinaryNumberException(binary);
                }
            }
                
            return integer;
        }`

but I felt like I've cheated on the homework so I decided to implement my own logic.